### PR TITLE
Css issues fixes

### LINKED ISSUE #938: CSS bug for table of contents
The TOC overlays the content for window widths of 992px - 1200px:

image

### PR DESCRIPTION
**Pull request by umgelurgel**
fixes #938  #964 

Also moves bootstrap and jquery dependencies to the repo and updates the new templates static urls.

image

image

### CODE REVIEW

File: templates/ep19/bs/base.html [hunk 1]
```diff
@@ -20,9 +20,9 @@
     {% endblock meta %}
 
     {# TODO: make it local link to boostrap #}
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link rel="stylesheet" href="/static/css/ep2019-theme.css">
-    <link rel="stylesheet" href="/static/css/ep2019-customisations.css">
+    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
+    <link rel="stylesheet" href="{% static 'css/ep2019-theme.css' %}">
+    <link rel="stylesheet" href="{% static 'css/ep2019-customisations.css' %}">
```
Comment (patrick91) [inline templates/ep19/bs/base.html:24]: can we merge the last two css files?

File: templates/ep19/bs/content/generic_content_page_with_sidebar.html [hunk 2]
```diff
@@ -42,3 +42,12 @@ <h3>Table of contents</h3>
 {% block morejs %}
 <script src="{% static "/js/generate_toc_for_cms.js" %}"></script>
 {% endblock morejs %}
+
+{% block morecss %}
+<style>
+    img.filer_image {
```
Comment (patrick91) [inline templates/ep19/bs/content/generic_content_page_with_sidebar.html:48]: 👍

File: templates/ep19/bs/footer/footer.html [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
+{% load menu_tags static %}
+
 <div id="footer" style='font-size: .9em'>
     <div class='container'>
         <div class='row'>
             {% show_menu 0 100 100 100 "ep19/bs/footer/_show_menu_in_footer.html" %}
 
             <div class='col-md-2'>
-                <a href='https://www.europython-society.org'><img src="/static/img/epslogo.png"></a>
+                <a href='https://www.europython-society.org'><img src="{% static 'img/epslogo.png' %}"></a>
```
Comment (patrick91) [inline templates/ep19/bs/footer/footer.html:9]: 👍

**Review by patrick91 — APPROVED** [review]
Just added a small note :)

File: templates/ep19/bs/base.html (hunk 1 above)
Comment (umgelurgel) [reply templates/ep19/bs/base.html:24]: `ep2019-theme` seems like a minified bootstrap (possibly just a subsection, the comments don't say - @artcz do you know?) the other seems to be manually-maintained file. We could probably retire either `ep2019-theme.css` completely, seems like it contains an older bootstrap version and is generally a smaller file
```
-rw-r--r-- 1 jakub jakub 155758 May 18 19:54 bootstrap.min.css
-rw-r--r-- 1 jakub jakub   3091 May  8 18:19 bs-stepper.min.css
-rw-r--r-- 1 jakub jakub   3211 May  8 18:19 ep2019-customisations.css
-rw-r--r-- 1 jakub jakub 121721 May  8 18:19 ep2019-theme.css
```

Comment (umgelurgel) [reply templates/ep19/bs/base.html:24]: Yeah, ep-2019 seems safe to remove. Thanks for pointing it out!

**Pull request merged**